Error when running tests

I'm running Node 6.2.1 on Windows 10. When running `npm test` on a fresh repo, I get the following error:

```
$ npm test

> prettier@0.0.2 test C:\Users\weeseng\Documents\GitHub\prettier
> jest

TypeError: string.replace is not a function
    at string (C:\Users\weeseng\Documents\GitHub\prettier\node_modules\jest-util\build\index.js:45:19)
    at p (C:\Users\weeseng\Documents\GitHub\prettier\node_modules\jest\node_modules\jest-cli\build\SearchSource.js:65:26)
    at new SearchSource (C:\Users\weeseng\Documents\GitHub\prettier\node_modules\jest\node_modules\jest-cli\build\SearchSource.js:101:34)
    at Runtime.createHasteContext.then.hasteMap (C:\Users\weeseng\Documents\GitHub\prettier\node_modules\jest\node_modules\jest-cli\build\jest.js:191:20)
    at process._tickCallback (internal/process/next_tick.js:103:7)
npm ERR! Test failed.  See above for more details.
```

Not too familiar with Jest, so I can't really help debug :(

--- a/package.json
+++ b/package.json
@@ -23,9 +23,7 @@
     "setupFiles": [
       "<rootDir>/tests_config/run_spec.js"
     ],
-    "testRegex": [
-      "jsfmt\\.spec\\.js$"
-    ],
+    "testRegex": "jsfmt\\.spec\\.js$",
     "testPathIgnorePatterns": [
       "tests/new_react",
       "tests/more_react"

--- a/tests_config/run_spec.js
+++ b/tests_config/run_spec.js
@@ -23,7 +23,7 @@ function run_spec(dirname) {
       const path = dirname + '/' + filename;
 
       if (!RUN_AST_TESTS) {
-        const source = read(path);
+        const source = read(path).replace(/\r\n/g, '\n');
         const output = prettyprint(source, path);
         test(filename, () => {
           expect(source + '~'.repeat(80) + '\n' + output).toMatchSnapshot();

